pkgs/by-name/ra/rav1e/package.nix

```diff
@@ -25,22 +22,19 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-Db7qb7HBAy6lniIiN07iEzURmbfNtuhmgJRv7OUagUM=";
   };
 
+  env.ZLIB_NO_VENDOR = "1";
+
   cargoHash = "sha256-VyQ6n2kIJ7OjK6Xlf0T0GNsBvgESRETzKZDZzAn8ZuY=";
 
   depsBuildBuild = [ pkg-config ];
 
   nativeBuildInputs = [
     cargo-c
-    libgit2
     nasm
+    zlib
```

Comment:
this seems wrong. I think it has something to do with `pkg-config` in `depsBuildBuild`. moving `pegs-config` to `nativeBuildInputs` makes things work as I'd expect -- `zlib` only required in `buildInputs`